src/main/java/edu/cornell/kfs/concur/ConcurConstants.java

```diff
@@ -147,5 +147,49 @@
         public static final String NOTIFICATION = "http://www.concursolutions.com/api/notification/2012/06";
         public static final String TRAVEL_REQUEST = "http://www.concursolutions.com/api/travelrequest/2012/06";
     }
+    
+    public static final class ConcurOAuth2 {
+        public static final class WebServiceCredentialKeys {
+            public static final String GROUP_CODE = "CNCROAUTH2";
+            public static final String CLIENT_ID = "concur.client_id";
+            public static final String SECRET_ID = "concur.secret_id";
+            public static final String USER_NAME = "concur.user_name";
```

Comment:
These values were what Concur sent us for the new application account, and we are required to keep track of them.  The webservice credential table seems like the bast place to keep track of this data.  I am not sure if the username will be needed for API calls.  However, it will be needed if we need Concur support to alter the account for us.